clients/drcachesim/tracer/raw2trace.h

```diff
@@ -1510,7 +1510,7 @@ class raw2trace_t {
     bool
     handle_rseq_abort_marker(raw2trace_thread_data_t *tdata,
```

Comment:
> Issue: https://github.com/DynamoRIO/dynamorio/issues/7084

Looks like you filed an issue covering just this mismatch: so this should be `Fixed: #7084`, right?
(Also note that with something like this where it was a PR that was just barely merged that had a small mistake, if you do a quick follow-up PR to fix it you can just use the same issue number (#7050 here) in the follow-up as was used in the main PR that introduced the small problem: no real benefit to filing a whole new issue that will be closed right after the other issue.)

Reply:
Ack.